src/index.js

```diff
@@ -0,0 +1,44 @@
+import './normalize.css';
+import './styles.css';
+import './styles.media.css';
+
+import createTextarea from './app/component/textarea/textarea';
+import createKeyboard from './app/component/keyboard/keyboard';
+import createElement from './app/utils/create-element';
+
+const CssClasses = {
+  BODY: 'wrap',
+  TITLE: 'wrap__title',
+  P: 'wrap__p',
+  TEXTAREA: 'wrap__textarea',
+  KEYBOARD: 'wrap__keyboard',
+};
+
+const textarea = createTextarea();
+textarea.classList.add(CssClasses.TEXTAREA);
+const lorem = 'Minim sunt eiusmod mollit dolore occaecat nulla ut id laboris veniam id. Eu culpa eiusmod ullamco cillum velit non dolor aliquip sit deserunt id est reprehenderit. Cillum esse tempor sint exercitation nisi exercitation est laboris.';
```

Comment:
It looks like reusable thing, we could put it into separate file with constants, or this could be utility function `generateLorem` etc.